packages/env/package.json

```diff
@@ -0,0 +1,48 @@
+{
+	"name": "@wordpress/env",
+	"version": "1.0.0",
```

Comment:
Two things here:
- Initial version should be lower than `1.0.0` because at the moment Lerna always bumps version.
- `.npmrc` file is missing

See more details:
https://github.com/WordPress/gutenberg/blob/master/packages/README.md#creating-a-new-package

Reply:
I'll add it to the follow up PR.